forwarder/kafka/kafka_producer.py

```diff
@@ -38,9 +39,13 @@ def ack(err, _):
                 topic, payload, key=key, on_delivery=ack, timestamp=timestamp_ms
             )
         except BufferError as e:
+            self._missed_data_count += 1
             self.logger.error(
                 "Producer message buffer is full. "
                 "Data loss occurred as messages are produced "
                 f"faster than are sent to the kafka broker: {e}"
             )
+            self.logger.error(
```

Comment:
May be combine this info in the previous log message. I am not sure.  What do you think?